[terminus_schedules] Add endpoint in the documentation and set data_fressness default value to 'base_schedule' (#3233)
CanalTP/navitia: documentation/slate/source/includes/apis.md

|additional_informations|[additional_informations](#additional-informations)|Other informations, when no departures<br> enum values:<ul><li>date_out_of_bounds</li><li>terminus</li><li>partial_terminus</li><li>active_disruption</li><li>no_departures_known</li></ul>|

<a name="terminus-schedules"></a>Terminus Schedules
---------------------------------------------------

>[Try it on Navitia playground (click on "EXT" buttons to see times)](http://canaltp.github.io/navitia-playground/play.html?request=https%3A%2F%2Fapi.navitia.io%2Fv1%2Fcoverage%2Fsandbox%2Fstop_areas%2Fstop_area%253ARAT%253ASA%253AGDLYO%2Fterminus_schedules%3Fitems_per_schedule%3D2%26&token=3b036afe-0110-4202-b9ed-99718476c2e0``` shell

``` shell
#request
$ curl 'https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/terminus_schedules' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'

#response

> Same as stop_schedule but objects embedded in terminus_schedules instead of stop_schedules

Reviewer: ```suggestion
Same as stop_schedule but objects are embedded in the `terminus_schedules` section instead
```

HTTP/1.1 200 OK
{
    "terminus_schedules": [],
    "pagination": {...},
    "links": [...],
    "disruptions": [],
    "notes": [],
    "feed_publishers": [...],
    "exceptions": []
}
```

Also known as `/terminus_schedules` service.

This endpoint gives you access to time tables going through a stop point.

> Departures are groupped observing all the stations being served after considered stop point. This can also be same as:<br>

Reviewer: ```suggestion
Departures are grouped observing all served stations after being considered stop point. This can also be  same as:<br>
```

Reviewer: I guess, the same apply down below
